pol-core/pol/accounts/account.cpp

```diff
@@ -66,7 +63,10 @@ void Account::readfrom( Clib::ConfigElem& elem )
 
   enabled_ = elem.remove_bool( "ENABLED", true );
   banned_ = elem.remove_bool( "BANNED", false );
-  uo_expansion_ = convert_uo_expansion( elem.remove_string( "UOExpansion", "T2A" ) );
+  auto exp = elem.remove_string( "UOExpansion",
+                                 Core::settingsManager.ssopt.features.expansionName().c_str() );
+  expansion_ = Plib::AccountExpansion(
+      exp, Core::settingsManager.ssopt.features.extensionFlags() );  // store flags?
```

Comment:
is this comment relevant? since the flags can be calculated from the expansion object?